tools/install.sh

```diff
@@ -1,3 +1,4 @@
+function main {
```

Comment:
Is your second comment about indenting the function body? I'm suggesting we not do that in this case, because it's just wrapping the whole file as a delayed-evaluation mechanism, and doing so would touch every line and have "git blame" point to this commit, which is not helpful.